.env

```diff
@@ -0,0 +1,5 @@
+USER=dockerci
+PASS=dockerci
+
+SSH_PORT=5022
+HTTP_PORT=80
```

Comment:
newline.

this isn't a posix file

Reply:
looks good I was meaning to add a `.env` file eventually.

Reply:
Sorry, I meant this file needs a newline, as a file defined in POSIX standards ends in a newline.